src/helpers/misc.js

```diff
@@ -48,6 +48,9 @@ const isValidETHAddress = address => {
 const isValidENSorEtherAddress = address => {
   return isValidETHAddress(address) || isValidENSAddress(address);
 };
+const isValidRNSorRSKAddress = address => {
```

Comment:
you no longer need a seperate function as `isValidETHAddress` will check the address against the chainID

Reply:
I'm sorry, I think I didn't get you. 
Will you implement an `isValidETHAddress` method which will check an address against a chainID? 
Thanks!

Reply:
this is already implmented `isValidETHAddress` will call `isAddress` which will check against the `chainID` :)

Reply:
thanks!